dockerfiles/theia-dev/build.sh

```diff
@@ -12,7 +12,29 @@ base_dir=$(cd "$(dirname "$0")"; pwd)
 
 
 DIR=$(cd "$(dirname "$0")"; pwd)
+init --name:theia-dev "$@"
+
 
+# Build gnu tar image
+GNUTAR_IMAGE_NAME="eclipse-che-theia-tar-build"
+build_tar_image() {
+  printf "Building image ${GNUTAR_IMAGE_NAME}${NC}..."
+  if docker build -t eclipse-che-theia-tar-build > docker-build-log 2>&1 -<<EOF
+  FROM alpine:3.11.3
```

Comment:
would prefer to not have alpine based process baked into theia-dev. Can we use UBI8 so we can keep it updated w/ latest CVE fies?

Or, how do you plan to replace this for downstream?

Reply:
I was wondering, is GNU tar is already there downstream ?

Reply:
@nickboldt I see some ubi8 images not updated as well
latest is 8.1-407 but I see a lot of images still using 8.1-398 as well